projects/eventonica-project.md

```diff
@@ -164,16 +199,29 @@ Users and a Events table.
 
 Switching back to Javascript, using Inquirer, implement code in `app.createNewUser` to ask for a new ficticious users name and age, or some other attribute. Then, display this information and save it to your Postgres database. Using your database client run a `SELECT` query against your Users table to verify that your users are successfully created.
 
+
 #### Part 4 - Eventful API AJAX
 
 Run the starter code in eventfulAPI.js and read about [eventful-node](https://www.npmjs.com/package/eventful-node) and [Eventful API](http://api.eventful.com/docs/events/search)
+
 Once you feel comfortable about understanding what the code is doing, commenting out the example search code and implement your custom search code in `app.searchEventful`.
 
+
 Use Inquier to:
 1. Ask app user to input a keyword to search on Eventful, example: "dancing", "fashion", etc.
 2. Display the first result that comes back from Eventful, and ask app user if he or she would like to save this to the Postgres database.
 3. If app user chooses yes, then make that save.
-4. If app user chooses no, then go back to step 1 of part 4 
+4. If app user chooses no, then go back to step 1 of part 4
```

Comment:
Why is this edit here?

Reply:
Maybe it is because some files were conflicting when I pushed it. So I resolved conflicts. There was a duplication of some lines, therefore, I deleted one line and kept one line as it is. 😊

Reply:
Unless the changes you’re resolving are yours and new, best practice is to keep the master version in a merge conflict. I only learned that recently.